src/Telemetry/Microsoft.DotNet.AzureDevOpsTimelineTests/Microsoft.DotNet.AzureDevOpsTimelineTests.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <SignAssembly>false</SignAssembly>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NUnit" />
```

Comment:
This assembly needs to be named right. If it ends with ".Tests", the arcade code will take over and fix all the test references.  Otherwise the test scanner won't find this assembly to run during PR's.

Reply:
👍